src/styles/stopwatch.scss

```diff
@@ -0,0 +1,47 @@
+
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+  &__minutes-hand {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform-origin: bottom center;
+    transform: translate(-50%, -100%) rotate(0deg);
+    animation: rotate2 600s steps(60) infinite;
```

Comment:
Ensure that the name of the keyframes 'rotate2' is descriptive enough to understand what kind of animation it represents. It's a good practice to use names that reflect the purpose or effect of the animation.